docs/source/conf.py

```diff
@@ -33,6 +38,7 @@
     "sphinx.ext.extlinks",
     "numpydoc",
     "sphinx_click.ext",
+    "yamltohtml",
```

Comment:
I might rename the extension file (and extension) to something more dask specific to match the new `dask-config-block`.